### DISCUSSION TITLE
Running a Vercel build locally

### DISCUSSION THREAD

**Discussion by Adam-Collier** [remix-run/remix#1197] [Q&A] [answered]
Hi guys, is there a way to run a vercel Remix build locally? I've noticed that the Remix App Server comes with an `npm start` script in the package.json however the vercel template doesnt seem to have this option

**Comment by Girish21** [comment, marked as answer]
you can run `vercel dev` which will spin up the vercel local dev server

**Reply by Adam-Collier** [reply]
Sweet thanks! Do I point it to `public` or `public/build`?

**Reply by Girish21** [reply]
Vercel CLI takes care of that for us :)
